crawl-ref/source/attack.h

```diff
@@ -13,8 +13,8 @@ const int HIT_WEAK   = 7;
 const int HIT_MED    = 18;
 const int HIT_STRONG = 36;
 
-const int BACKLIGHT_TO_HIT_BONUS = 12;
-const int UMBRA_TO_HIT_MALUS = -8;
+const int BACKLIGHT_TO_HIT_BONUS = 5;
+const int UMBRA_TO_HIT_MALUS = -3;
```

Comment:
Hm, wasn't this a lot smaller than the old values effectively were? Or am I misremembering?

Reply:
Original code was:
`        if (defender->backlit(false))
            mhit += 2 + random2(8);
        else if (!attacker->nightvision()
                 && defender->umbra())
            mhit -= 2 + random2(4);`
Means of 5.5 and 3.5, so we can't precisely model them with a constant. I just used the same constant values that you'd used in the attack.cc code, which was rounding them both down.